### PR TITLE
Template project

### PR DESCRIPTION
**Pull request by rt1301**
I have added dat.gui library functions to most of the art pieces. This gives the user ability to play around with some parts of the code and see the corresponding changes immediately.
Screenshot of Change:
Screenshot from 2020-12-20 17-03-13
Screenshot from 2020-12-20 17-03-39

### CODE REVIEW

File: src/components/ArtDisplay/art-pieces/0_Template/sketch.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        p.strokeWeight(2);
+        p.noLoop();
+        p.frameRate(2);
+    }
+    // draw() runs after setup(), normally on a loop
+    // in this case it runs only once, because of noLoop()
+    p.draw = () => {
+        p.background(255);
+        for (let y = padding; y < p.height - 2 * padding; y += cellSize + gap) {
+          for (let x = padding; x < p.width - 2 * padding; x += cellSize + gap) {
```
Comment (Jayashrri) [inline src/components/ArtDisplay/art-pieces/0_Template/sketch.js]: `gap` hasn't been defined anywhere. This is raising an error.

File: src/components/ArtDisplay/art-pieces/PyramidGrid/sketch.js [hunk 2]
```diff
@@ -96,4 +96,38 @@ export default function sketch(p) {
...
+  const uiShade = gui.addColor(pyramidGrid, "Shade");
+  const uiRow = gui.add(pyramidGrid, "row", 1, 6, 1);
+  uiShiftLimit.onChange(() => {
+    p.setup();
+    p.draw();
+  });
+  uiFrameRate.onChange(() => {
+    p.setup();
+    p.draw();
+  });
```
Comment (Jayashrri) [inline src/components/ArtDisplay/art-pieces/PyramidGrid/sketch.js:120]: This is not getting animated.

File: src/components/ArtDisplay/art-pieces/HazyPyramids/sketch.js [hunk 3]
```diff
@@ -126,4 +126,35 @@ export default function sketch(p) {
...
+  });
+  uiRow.onChange(() => {
+    p.setup();
+    p.draw();
+  });
+  uiShade.onChange(() => {
+    p.setup();
+    p.draw();
+  });
+  uiFrameRate.onChange(p.setup);
```
Comment (Jayashrri) [inline src/components/ArtDisplay/art-pieces/HazyPyramids/sketch.js:155]: The frame rate doesn't seem to change on updating the selected value. Could you check once?

File: src/components/ArtDisplay/art-pieces/BeyondBinary/sketch.js [hunk 4]
```diff
@@ -69,20 +67,43 @@ export default function sketch(p) {
...
     }
     p.pop();
   }
+  const BeyondBinary = function () {
+    this.Color1 = "#EFCF20";
+    this.Color2 = "#A083D5";
+    this.Time1 = 3;
+    this.Time2 = 3;
+    this.frameRate = 1;
+  };
```
Comment (Jayashrri) [inline src/components/ArtDisplay/art-pieces/BeyondBinary/sketch.js]: You've used classes in the template, use that here too for uniformity?

**Review by Jayashrri — CHANGES_REQUESTED** [review]

File: src/components/ArtDisplay/art-pieces/PyramidGrid/sketch.js (hunk 2 above)
Comment (rt1301) [reply src/components/ArtDisplay/art-pieces/PyramidGrid/sketch.js:120]: This is actually a static drawing, hence you are not able to see any animation on changed framerate.

**Comment by rt1301** [comment]
@Jayashrri I have made the above-asked changes, please review them once. Thanks!

**Review by Jayashrri — APPROVED** [review]

**Pull request merged**